guillotina_s3storage/storage.py

```diff
@@ -394,3 +392,11 @@ async def iterate_bucket(self):
             ):
                 for item in result.get("Contents", []):
                     yield item
+
+    def _get_bucket_kargs(self, bucket_name: str):
+        bucket_kwargs: Dict[str, Any] = {"Bucket": bucket_name}
+        if self._get_region_name() != "us-east-1":
+            bucket_kwargs["CreateBucketConfiguration"] = {
+                "LocationConstraint": self._get_region_name(),
+            }
```

Comment:
What's the reason for not always adding the Location?
I guess "us-east-1" is the default, but it seems like good practice to always indicate the region.
Just curious.

Reply:
This is imo a weird choice from aiobotocore creators. Essentially you need to include the LocationConstraint if you are not using the default AWS region or you'll get an error when creating the bucket.

You can see the same pattern here:

https://github.com/aio-libs/aiobotocore/blob/4c18d3da3f34f3c79eda9c66e770ce8e8229adf1/tests/conftest.py#L352

I guess it makes no harm to include the constraint always